src/lib/components/Cytoscape.react.js

```diff
@@ -953,6 +962,25 @@ Cytoscape.propTypes = {
          */
         target: PropTypes.object,
     }),
+
+    /**
+     * Dictionary specifying configuration options to overlay a leaflet map on top of Cytoscape. All
+     * configuration options are optional; provide an empty dictionary to use default options. Requires
+     * latitude and longitude properties to be included in the node data for positional information. Provided node
+     * "id" will be used as the node label. Requires preset layout to be used. Requires extra layouts to be loaded.
+     */
+    leaflet: PropTypes.exact({
+        /** Endpoint used by leaflet to fetch map tiles. */
+        tileUrl: PropTypes.string,
+        /** Attribution text displayed on the bottom right corner of the map. */
+        attribution: PropTypes.string,
```

Comment:
Is there a required attribution to use with the default `tileUrl`? If so we should include that (if tileUrl was omitted or matches the default) so our users are encouraged to follow the rules.

More generally, is there a set of presets we can build in, like we do in [plotly.js mapbox](https://github.com/plotly/plotly.js/blob/bc66a584d17adf2f48aea7230d9f82b6a2a407bf/src/plots/mapbox/constants.js#L30-L159)? Looks like the format is a little different, but if you could set `leaflet.preset` to one of an enumerated set of strings, we could use this to set `tileUrl`, `attribution`, and `maxZoom` - subject to the user overriding any one of these, like:
```
leaflet={'preset': 'carto-voyager', 'maxZoom': 16}
```

Reply:
> Is there a required attribution to use with the default `tileUrl`? If so we should include that (if tileUrl was omitted or matches the default) so our users are encouraged to follow the rules.

This is automatically set already on the ext side in that (default) case.  We can have the component delegate to the extension when the consumer doesn't specify a custom tileset, unless the Dash component would benefit from a different default.

> 
> More generally, is there a set of presets we can build in, like we do in [plotly.js mapbox](https://github.com/plotly/plotly.js/blob/bc66a584d17adf2f48aea7230d9f82b6a2a407bf/src/plots/mapbox/constants.js#L30-L159)? Looks like the format is a little different, but if you could set `leaflet.preset` to one of an enumerated set of strings, we could use this to set `tileUrl`, `attribution`, and `maxZoom` - subject to the user overriding any one of these, like:
> 
> ```
> leaflet={'preset': 'carto-voyager', 'maxZoom': 16}
> ```

Makes sense.  Same list?

Reply:
> Same list?

sure, if they all work the same here!

Reply:
All Leaflet preset providers are now supported